src/vectorclock.erl

```diff
@@ -68,6 +114,51 @@ max([]) -> new();
 max([V]) -> V;
 max([V1,V2|T]) -> max([merge(fun erlang:max/2, V1, V2)|T]).
 
+-spec max_vc(vectorclock(), vectorclock()) -> vectorclock().
```

Comment:
The cases a≤b and b≤a are clear and for these cases `max([a,b])` should give the same result. But for concurrent clocks your implementation always returns the first argument. 

Maybe call it `total_order_max`, since it calculates the max, assuming the two clocks are totally ordered? Or call it `replace_if_greater` since it replaces the first clock with the second, if the second is greater.